### DISCUSSION TITLE
How to use this with an third party wiimote??

### DISCUSSION THREAD

**Discussion by yumnycheesecakes** [v1993/linuxmotehook2#21] [Q&A]
Hi everyone, recently I bought an third party wiimote. It gets paired with my PC, but Dolphin doesn't pick it up. So is there an way to use this with an third party wiimote?? This software is my last hope.

**Comment by v1993** [comment]
The procedure is identical to using an official wiimote. I'd suggest trying passthrough mode in dolphin first, though.

**Reply by yumnycheesecakes** [reply]
Sadly passthrough isn't my way because I need to play with my friends, and I have only one Wii Remote. How do I get the ID of the Wii Remote to write in the config??

**Reply by v1993** [reply]
You can run without a configuration file to get started.

**Reply by yumnycheesecakes** [reply]
I don't know if it's just mine problem, but the Wii Remote is still blinking, and there's no DSU devices from Dolphin, I have added the DSU server. The Wii Remote is connected in Bluetooth though

**Reply by v1993** [reply]
It's likely not picked up by the kernel driver, then. You can try running `sudo dmesg --follow` and see what shows up in console on wiimote connection. Sadly, it seems fairly likely that linuxmotehook2 won't work for you.

**Reply by yumnycheesecakes** [reply]
```
[11959.622144] Bluetooth: hci2: Opcode 0x2005 failed: -110
[11961.752084] Bluetooth: hci2: Opcode 0x200b failed: -110
[12147.708784] Bluetooth: hci2: unexpected cc 0x1405 length: 3 < 4

```
Maybe it's because the bluetooth driver??

**Reply by v1993** [reply]
That's possible but honestly, hard to say. Those errors correspond to commands timing out, so it may be the wiimote as well.

**Reply by yumnycheesecakes** [reply]
Is there any solutions?? That Wiimote was kinda expensive...

**Reply by v1993** [reply]
I'd still recommend trying wiimote passthrough in dolphin, since many games support sharing a single wiimote between multiple players taking turns. Maybe see if it's a permissions issue if that fails (try `sudo chmod a+rw /dev/hidraw*`).

Aside from that, there sadly isn't much more to try short of getting a different bluetooth adapter or a first-party wiimote.

**Reply by yumnycheesecakes** [reply]
I live in Russia, aside of Wii getting released there it's still hard to find original Wiimotes there. So I guess I have to get an another bluetooth adapter.

**Reply by yumnycheesecakes** [reply]
I will try the chmod method later.

**Reply by yumnycheesecakes** [reply]
Okay, is there an tutorial on how to set this up properly?? I think I have missed something
